datacube/utils/geometry.py

```diff
@@ -422,7 +422,7 @@ def buffer(self, distance, quadsecs=30):
     def simplify(self, tolerance):
         return _make_geom_from_ogr(self._geom.Simplify(tolerance), self.crs)
 
-    def to_crs(self, crs, resolution=None):
+    def to_crs(self, crs, resolution=None, wrapdateline=False):
```

Comment:
It'd be good to get some documentation here describing what this does.

Mention that it only makes sense for geographic (maybe even throw an exception if passing it for a non-geographic projection.) And I assume that it turns a Polygon into a MultiPolygon with a piece on each side...